tests/capture.spec.js

```diff
@@ -0,0 +1,33 @@
+// @ts-check
+import { test, expect } from "@playwright/test";
+
+let adminApiContext;
+let page;
+
+
+test.beforeAll(async ({ playwright, baseURL, browser }) => {
+    page = await browser.newPage();
+    await page.waitForTimeout(1000);
+    await page.goto("/overlay");
+    adminApiContext = await playwright.request.newContext({
+        baseURL: `${baseURL}/api/admin/`
+    });
+});
+
+test.afterAll(async () => {
+    await page.close();
+});
+
+const simpleWidgets = ["scoreboard", "statistics"];
+
+for (const widgetName of simpleWidgets) {
+    test(`test ${widgetName}`, async () => {
```

Comment:
This works for now, but we need to think of a way to structure the tests in the playwright paradigm.